src/languages/sqf.js

```diff
@@ -2147,7 +2269,7 @@ export default function(hljs) {
     'showCommandingMenu',
     'showCompass',
     'showCuratorCompass',
-    'showGPS',
+    'showGps',
```

Comment:
```suggestion
    'showGPS',
```
https://community.bistudio.com/wiki/showGPS